src/main/java/core/basesyntax/service/operation/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    void handle(FruitTransaction fruitTransaction, Storage fruitStorage);
```

Comment:
Just call static methods of the `Storage` class in each handler
```suggestion
    void handle(FruitTransaction fruitTransaction);
```